src/toast/schedule_sim_ground.py

```diff
@@ -2654,7 +2652,7 @@ def parse_args(opts=None):
     --patch name,weight,lon,lat,radius (center and radius)
     --patch name,weight,lon_min,lat_max,lon_max,lat_min (rectangle)
     --patch name,weight,lon1,lat1,lon2,lat2,...,lonN,latN (polygon, N>=3)
-    --patch name,SSO,weight,radius (Solar System Object)
+    --patch name,SSO,weight,radius,eta_off,xi_off (Solar System Object, last two optional offset for pointingnat specific place on fp)
```

Comment:
Can you add square brackets around the optional fields: `,eta_off,xi_off` => `[,eta_off,xi_off]` so it is immediately obvious they are optional?

There is also a missing space at `pointingat`.